src/javascript_bindings.pyx

```diff
@@ -29,14 +29,15 @@ cdef class JavascriptBindings:
     cpdef py_void SetFunction(self, py_string name, object func):
         self.SetProperty(name, func)
 
-    cpdef py_void SetObject(self, py_string name, object obj):
+    cpdef py_void SetObject(self, py_string name, object obj,
+                            py_bool allow_properties=False):
```

Comment:
The function name could be changed to GetObjectAttribute but that might just cause conflicts for existing users. Maybe mirror it with a new function of that name and mark the current function deprecated?